### ISSUE TITLE
Fix CSP errors on homepage for brackets

### ISSUE THREAD

**Issue by gideonthomas** [mozilla/thimble.mozilla.org#1394]
```
[Report Only] Refused to connect to 'https://mozillathimblelivepreview.net/bramble/production/dist/styles/brackets.min.css' because it violates the following Content Security Policy directive: "connect-src 'self'".
Y.cors.e.crossDomain.send @ /homepage/scripts/main.js:3
/homepage/scripts/main.js:3 [Report Only] Refused to connect to 'https://mozillathimblelivepreview.net/bramble/production/dist/bramble.js' because it violates the following Content Security Policy directive: "connect-src 'self'".
Y.cors.e.crossDomain.send @ /homepage/scripts/main.js:3
/homepage/scripts/main.js:3 [Report Only] Refused to connect to 'https://mozillathimblelivepreview.net/bramble/production/dist/main.js' because it violates the following Content Security Policy directive: "connect-src 'self'".
Y.cors.e.crossDomain.send @ /homepage/scripts/main.js:3
/homepage/scripts/main.js:3 [Report Only] Refused to connect to 'https://mozillathimblelivepreview.net/bramble/production/dist/thirdparty/thirdparty.min.js' because it violates the following Content Security Policy directive: "connect-src 'self'".
```

Since we XHR brackets, we need to add the bramble uri to our CSP config for `connect-src`.
The code change basically needs to happen here: https://github.com/mozilla/thimble.webmaker.org/blob/bramble/server/index.js#L81-L84

**Comment by yash14123** [comment]
I would like to take this up.

**Comment by gideonthomas** [comment]
sure @yash14123! You basically just need to pass is a property called `connection` with the value identical to the other properties passed in the code linked above. Let's see if that works!

**Comment by TridevGuha** [comment]
is this bug #1394 fixed?

**Comment by gideonthomas** [comment]
@TridevGuha no, would you like to take this on?

**Comment by TridevGuha** [comment]
yes I would, help me with the first steps please.

**Comment by gideonthomas** [comment]
Fixed in 7347221b4affad4701693f5d2e8ed03b35bfbdab
